app/index.html

```diff
@@ -5,9 +5,13 @@
     <title></title>
     <meta name="description" content="">
     <meta name="viewport" content="width=device-width">
+
+    <link rel="stylesheet" href="https://cartodb-libs.global.ssl.fastly.net/cartodb.js/v3/3.14/themes/css/cartodb.css" />
```

Comment:
For this and the js src below, should probably use the official CDN at `libs.cartocdn.com`.